fortran/util.F90

```diff
@@ -10,7 +10,14 @@ module musica_util
   private
 
   public :: string_t_c, string_t, error_t_c, error_t, mapping_t_c, mapping_t, &
-            to_c_string, to_f_string, assert, copy_mappings, delete_string_c, create_string_c
+            to_c_string, to_f_string, assert, copy_mappings, delete_string_c, &
+            create_string_c, musica_rk, musica_dk
+
+  !> Single precision kind
+  integer, parameter :: musica_rk = kind(0.0)
+
+  !> Double precision kind
+  integer, parameter :: musica_dk = kind(0.d0)
```

Comment:
`rk` and `dk` here refer to the `real` and `double precision` type in Fortran. They can be any name (In CAM, we use `r4` for `real` and `r8` for `double`).

Reply:
I think they stand for "real kind" and "double kind", but I don't think there is any real convention for these (or anything else) in Fortran. I'm fine with any names for these, if there's something that makes it more clear what they are.

I typically alias these in modules where they're used because they end up in every variable declaration and floating point literal, and long names get a little difficult to work with:

```
use musica_util, only : dk => musica_dk
real(dk) :: foo
foo = 1.2_dk
```

Reply:
(sorry for the duplicate info, just saw @sjsprecious 's comment)

Reply:
Ah I see, I am fine with the acronym. Thank you both for the answer!